File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = []
    count1 = 0
    count2 = 0
    for i in num_list:
        if i % 2 == 0:
            count1+=1
        else:
            count2+=1
            
    answer = [count1, count2]
    return answer

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = []
    let count1 = 0
    let count2 = 0
    for (let i in num_list) {
        if (num_list[i] % 2 == 0) {
            count1++
        }
        else {
            count2++
        }
    }
    answer[0] = count1
    answer[1] = count2
    return answer;
}